api/lds.proto

```diff
@@ -66,13 +66,27 @@ message FilterChainMatch {
 
   // If non-empty, an IP address and prefix length to match addresses when the
   // listener is bound to 0.0.0.0/::.
```

Comment:
Are you suggesting that if the original destination is IP X but we receive it on a Listener bound to IP Y that we use the FilterChainMatch table for IP Y to determine the filter chain for IP X, rather than going back to the Listeners and trying to find a Listener matching IP X (which might also exist)?